backend/cn/main.ml

```diff
@@ -217,6 +228,7 @@ let main
       print_log_file ("mucore", MUCORE prog5);
       let paused = Typing.run_to_pause Context.empty (Check.check_decls_lemmata_fun_specs prog5) in
       Result.iter_error handle_error (Typing.pause_to_result paused);
+      maybe_generate_tests output_test_dir filename test_max_depth testing_framework ail_prog prog5;
       begin match output_decorated with
       | None ->
         Typing.run_from_pause (fun paused -> Check.check paused lemmata) paused
```

Comment:
This will run the proof checking after generating tests, which is likely not what we want.

Reply:
I figured that could be dealt with after we split things into subcommands...

Reply:
Though I suppose for CI tests, why waste the time.